src/pages/home/report/ReportActionCompose/AttachmentPickerWithMenuItems.js

```diff
@@ -152,7 +152,7 @@ function AttachmentPickerWithMenuItems({
      */
     const taskOption = useMemo(() => {
         // We only prevent the task option from showing if it's a DM and the other user is an Expensify default email
-        if (!Permissions.canUseTasks(betas) || ReportUtils.isExpensifyOnlyParticipantInReport(report)) {
+        if (!Permissions.canUseTasks(betas) || (!ReportUtils.isPolicyExpenseChat(report) && ReportUtils.isExpensifyOnlyParticipantInReport(report))) {
```

Comment:
What's the purpose of adding the extra check for workspace chats here?

Reply:
Well I noticed that the assign task option wasn't present in workspace chats that were owned by us. So that was inconsistent with other workspace chats. But I should also make this check similar to the money request one where it is based on who the policy owner is.

Reply:
Ok I moved this logic into a function so it should be clearer.